qiskit_nature/operators/second_quantization/spin_op.py

```diff
@@ -96,12 +96,14 @@ class SpinOp(SecondQuantizedOp):
 
         "X_0"
         "Y_0^2"
-        "Y_1^2 Z_1^3 X_0^1 Y_0^2 Z_0^2"
+        "Y_0^2 Z_0^3 X_1^1 Y_1^2 Z_1^2"
 
     are possible labels.
     For each :code:`index` the operations `X`, `Y` and `Z` can only be specified exclusively in
-    this order. `+` and `-` are same order with `X` and cannot be used with `X` and `Y`.
+    this order. `+` and `-` are same order with `X` and cannot be used with `X` and `Y`
```

Comment:
I think I don't fully understand what you mean by `+ and - are same order with X`..
Do you mean they have to be located in the word ordering where `X` would be?
But you cannot combine them with `X` and `Y` anyways right?
Can you combine them with `Z`? And in that case, do they need to occur before `Z`?
Finally, do you allow `-_0 +_0` or only `+_0 -_0`?

Reply:
> Can you combine them with Z? And in that case, do they need to occur before Z?

Yes. `SpinOp("+_0 Z_0", register_length=1)` is valid, but `SpinOp("Z_0 +_0", register_length=1)` is invalid.

> Finally, do you allow -_0 +_0 or only +_0 -_0?

only `+_0 -_0`

I think `same order` is difficult to understand, so simplly I will remove the phrase.

Reply:
```suggestion
    this order. `+` and `-` cannot be used with `X` and `Y`
```

Reply:
Yes that sounds good! Please go ahead with this :+1: